x/bep3/client/cli/query.go

```diff
@@ -293,8 +327,14 @@ $ kvcli q bep3 swaps --page=2 --limit=100
 				return fmt.Errorf("No matching atomic swaps found")
 			}
 
+			formattedAtomicSwaps := []FormattedAtomicSwap{}
+
+			for _, swap := range matchingAtomicSwaps {
+				formattedAtomicSwaps = append(formattedAtomicSwaps, NewFormattedAtomicSwap(swap))
+			}
+
 			cliCtx = cliCtx.WithHeight(height)
-			return cliCtx.PrintOutput(matchingAtomicSwaps.String()) // nolint:errcheck
+			return cliCtx.PrintOutput(formattedAtomicSwaps)
```

Comment:
I just matched the existing output of the CLI command, but it doesn't have parity with the REST API.

Are we OK with that?

Example:
CLI YAML
```
ID: FD6C531A2E0B8AF67030E59D0ED2424E3380950DB7444FC1CEB30D8369F83057
amount: 19999000bnb
random_number_hash: 1b24c223bb9e59385416dbcf71639efdb401dd9a14159c57ee704a8fd766b43b
expire_height: 72998
timestamp: 1592285967
sender: kava1r4v2zdhdalfj2ydazallqvrus9fkphmglhn6u6
recipient: kava1gzc54u4p67hh9r4m9vcml3ke9fc29tplvvaev3
sender_other_chain: bnb1nv2fnfc6xvvve2qvtzg2x5vp4h6wxt243d4ddl
recipient_other_chain: bnb1jh7uv2rm6339yue8k4mj9406k3509kr4wt5nxn
closed_block: 76556
status: Completed
cross_chain: true
direction: Incoming
```
CLI JSON
```
{
    "ID": "FD6C531A2E0B8AF67030E59D0ED2424E3380950DB7444FC1CEB30D8369F83057",
    "amount": "19999000bnb",
    "random_number_hash": "1b24c223bb9e59385416dbcf71639efdb401dd9a14159c57ee704a8fd766b43b",
    "expire_height": "72998",
    "timestamp": "1592285967",
    "sender": "kava1r4v2zdhdalfj2ydazallqvrus9fkphmglhn6u6",
    "recipient": "kava1gzc54u4p67hh9r4m9vcml3ke9fc29tplvvaev3",
    "sender_other_chain": "bnb1nv2fnfc6xvvve2qvtzg2x5vp4h6wxt243d4ddl",
    "recipient_other_chain": "bnb1jh7uv2rm6339yue8k4mj9406k3509kr4wt5nxn",
    "closed_block": "76556",
    "status": "Completed",
    "cross_chain": true,
    "direction": "Incoming"
}
```
REST JSON
```
{
      "amount": [
        {
          "denom": "bnb",
          "amount": "9999000"
        }
      ],
      "random_number_hash": "6AB6AC1A5DB9CB2EDF782D1D5D2076A9BBCCC663F8CB8D8CE11366778E77B74C",
      "expire_height": "59101",
      "timestamp": "1592193199",
      "sender": "kava1r4v2zdhdalfj2ydazallqvrus9fkphmglhn6u6",
      "recipient": "kava1qa5tpxdy9r636nt2ze76g8k5h6zce807z4tksg",
      "sender_other_chain": "bnb17sd249m4mtr66n3ux057rvjs0c09sqf5rvq39n",
      "recipient_other_chain": "bnb1jh7uv2rm6339yue8k4mj9406k3509kr4wt5nxn",
      "closed_block": "58982",
      "status": "Completed",
      "cross_chain": true,
     "direction": "Incoming"
}
```